sysdig/data_source_sysdig_secure_trusted_cloud_identity.go

```diff
@@ -0,0 +1,50 @@
+package sysdig
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceSysdigSecureTrustedCloudIdentity() *schema.Resource {
+	timeout := 5 * time.Minute
+
+	return &schema.Resource{
+		ReadContext: dataSourceSysdigSecureTrustedCloudIdentityRead,
+
+		Timeouts: &schema.ResourceTimeout{
+			Read: schema.DefaultTimeout(timeout),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"cloud_provider": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
Thanks for the PR @nkraemer-sysdig :bow:! This is currently merged, but let me annotate a small improvement I saw for the future.

This field `cloud_provider` is required and valid values are `gcp`, `aws` and `azure`, as specified by https://github.com/sysdiglabs/terraform-provider-sysdig/pull/113/files#diff-5f4ddefaaf5477507c4eb64c17871c8ff9a97219c398a1b97a3735adf192bd7aR25
Currently I believe this is being validated by the backend, but we could also generate validations in the provider, so if an inconsistent value is provided in the module, the request is not performed to the backend: https://github.com/sysdiglabs/terraform-provider-sysdig/blob/91a7daf343569bbc3a67ced1d4d079854e105ed0/sysdig/resource_sysdig_secure_policy.go#L61

Reply:
That's a great idea! I will add this in the next PR I make

Reply:
Added in https://github.com/sysdiglabs/terraform-provider-sysdig/pull/115